src/lib/button-toggle/button-toggle.ts

```diff
@@ -65,13 +65,13 @@ export class MdButtonToggleGroup implements AfterViewInit, ControlValueAccessor
   private _name: string = `md-button-toggle-group-${_uniqueIdCounter++}`;
 
   /** Disables all toggles in the group. */
-  private _disabled: boolean = null;
+  private _disabled: boolean | null = null;
```

Comment:
It's because the `disabled` attribute needs to be removed from the DOM completely when enabled. We could also switch the bindings to be something like `disabled || null`.